### DISCUSSION TITLE
Pureartist version?? What should we do?

### DISCUSSION THREAD

**Discussion by bugswriter** [Bugswriter/tuxi#141] [Q&A]
If you guys haven't noticed we have one more branch pureartist-version. 
with just one developer @pureartistry
Recently I am having exams so .. I am just merging PR in pureartist-version branch. 
It's a very different implementation of our tuxi. But losing a little bit simplicity. 
I want your opinion what should I do. 
I mean obviously without proper understanding of code and without proper testing I am not gonna merge with main.
We have to look it's extensibility, even though it contain functions seems very extensible. 
Why don't you guys try to look at pureartist-version branch and tell me If you able to contribute to that. 
and if the guy who is working on this (seems like a pro) did any mistake. Anything important he missed? 

BTW we should all appreciate @PureArtistry for so many cool stuff he did.

**Comment by PureArtistry** [comment]
if anyone has any questions about the code or suggestions for it etc, just @me, happy to help

**Comment by Genghius** [comment]
replace development with it, once we get a chance to merge with main, do so.

**Reply by sudocanttype** [reply]
Gonna have to disagree with you here. There are a bunch of features (and bugs) in the dev branch that will probably become useful to implement in the future, such as the new set opts. Having a version checker, debug mode, etc would be cool.

**Reply by PureArtistry** [reply]
I have already implemented everything from the current dev branch (plus all the extra snippets from the PR list)
The only thing in dev I haven''t added was the version info but I can add that in now, will only take a sec

**Reply by BeyondMagic** [reply]
Develop is replaceable then after #143 merges.

**Reply by PureArtistry** [reply]
I'll be making one more commit to that in a minute to fix the weather bug for people who live in countries that use frht by default

**Reply by Genghius** [reply]
> Gonna have to disagree with you here. There are a bunch of features (and bugs) in the dev branch that will probably become useful to implement in the future, such as the new set opts. Having a version checker, debug mode, etc would be cool.

i cant disagree with that caus i have not even looked at dev branch since panacea 3
